common/achievement/types.ts

```diff
@@ -111,7 +111,7 @@ export type EvaluationResult = {
     resultObject: Record<string, string | number | boolean>;
 };
 
-export type RelationContextType = {
+export type AchievementContextType = {
     match?: {
         id: number;
         lecture: {
```

Comment:
Here I think we need an array of lectures or? So that we can create a bucket for each match appointment or?